usr/lib/raspberrypi-net-mods/mv-wpa.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+if [ -f "/boot/wpa_supplicant.conf" ]; then 
+  /bin/mv /boot/wpa_supplicant.conf /etc/wpa_supplicant/wpa_supplicant.conf
+  if [ -f "/boot/wpa_supplicant.conf.txt" ]; then 
+    /bin/rm /boot/wpa_supplicant.conf.txt
+  fi
+elif [ -f "/boot/wpa_supplicant.conf.txt" ]; then 
+  /bin/mv /boot/wpa_supplicant.conf.txt /etc/wpa_supplicant/wpa_supplicant.conf
+fi
+dos2unix /etc/wpa_supplicant.conf
```

Comment:
Typo? The previous lines move it to `/etc/wpa_supplicant/wpa_supplicant.conf` whereas this line is using `/etc/wpa_supplicant.conf`.

Reply:
Fixed